index.html

```diff
@@ -9,6 +9,13 @@
     specStatus: "ED",
     edDraftURI: "https://w3c.github.io/server-timing/",
     editors: [{
+      name: "Charles Vazac",
+      url: "???",
+      mailto: "cvazac@gmail.com",
+      company: "Akamai",
+      companyURL: "https://www.akamai.com/",
+      w3cid: "???"
```

Comment:
`w3cid: 77116`
for `url`, you can list `https://twitter.com/vazac` or leave blank, please